resources/dicts/patrols/beach/hunting/leaf-bare.json

```diff
@@ -2033,19 +2033,19 @@
                 "prey": ["very_small"]
             },
             {
-                "text": "Perhaps hunting from the sides of a slippery, frosty rock pool wasn't their best moment, r_c concedes. The patrol has to hurry back to camp to warm {PRONOUN/r_c/object} up.",
+                "text": "Perhaps hunting from the sides of a slippery, frosty rock pool wasn't their best moment, m_c concedes. The patrol has to hurry back to camp to warm {PRONOUN/m_c/object} up.",
```

Comment:
Within patrols, the only text that should utilize "m_c" is the history text.  The patrol text itself needs to remain as "r_c".  I know it's a bit confusing, it's just an unfortunate side effect of two systems being built at different times.  You'll need to go through this PR and undo all the m_c changes that aren't within history text.

Reply:
Okay, got it.